### PR TITLE
Improve content with a focus on functionality

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: specification/components/examples/ResponseFullPermissions.yaml [hunk 1]
```diff
@@ -1,4 +1,11 @@
-summary: Full permissions to medical record and services.
+summary: |
+  Patient has full permission to the current medical record, the current
+  medical record starts from the `effectiveDate` (1/11/2022). Access to
+  historical medical record is currently set to `summary` and limited to
```
Comment (sammydcruz) [inline specification/components/examples/ResponseFullPermissions.yaml:4]: where it says "access to historical medical record is currently set to "summary" and limited to detailed" I think the "limited to detailed" sounds confusing, like it is summary or detailed? Maybe needs to use the words maximum requestable or similar

File: specification/components/examples/ResponseRequestForDocumentsAccess.yaml [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+  A pending response from a request where the patient asked for detailed access
+  to the historical medical record.
 value:
   permissionType: medicalRecord
   subType: historical
-  accessLevel: documents
+  accessLevel: detailed
   status: pending
   message: |
     Request accepted to update access historical effective date to documents.
```
Comment (sammydcruz) [inline specification/components/examples/ResponseRequestForDocumentsAccess.yaml:10]: does this message need to be updated? it sounds a bit like the requested change has been accepted

File: specification/components/examples/ResponseRequestForDocumentsAccess.yaml [hunk 3]
```diff
@@ -1,9 +1,10 @@
 summary: |
-  Request for documents access for historical permissions that is pending.
+  A pending response from a request where the patient asked for detailed access
```
Comment (sammydcruz) [inline specification/components/examples/ResponseRequestForDocumentsAccess.yaml:2]: Maybe could be reworded "A response from a request where the patient asks for detailed access to the historical medical record. Response indicates the request is pending review"

File: specification/components/examples/ResponseRequestForDocumentsAccess.yaml (hunk 2 above)
Comment (st3v3nhunt) [reply specification/components/examples/ResponseRequestForDocumentsAccess.yaml:10]: Yeah, this message is a bit confused!

**Review by sammydcruz — APPROVED** [review]

**Review by sammydcruz — APPROVED** [review]

**Pull request merged**